### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by Pramod0215**

### CODE REVIEW

File: reddit_news/reddit_news/settings.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'reddit_app',
```
Comment (varunpsr) [inline reddit_news/reddit_news/settings.py]: not a valid app name. Please check naming conventions online and change.

File: reddit_news/reddit_app/models.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class User(models.Model):
+    user_name = models.CharField(max_length=25,unique=True)
+    email = models.EmailField(unique=True)
+
+    def __str__(self):
+        return self.user_name
+
+
+
+class Community(models.Model):
```
Comment (varunpsr) [inline reddit_news/reddit_app/models.py]: Please make a separate app for Post and Community.

File: reddit_news/reddit_news/settings.py (hunk 1 above)
Comment (Pramod0215) [reply reddit_news/reddit_news/settings.py]: changed to redditapp

File: reddit_news/reddit_app/models.py (hunk 2 above)
Comment (Pramod0215) [reply reddit_news/reddit_app/models.py]: done

**Pull request merged**